src/Command/DeleteServiceEventCommand.php

```diff
@@ -51,14 +52,14 @@ protected function executeJsonEvent(array $payload): ?array
     }
 
     /**
-     * @param string $elementToDel
+     * @param string[] $elementToDel
      * @param bool $default
      * @return ConfirmationQuestion
      */
-    private function getDeleteConfirmationQuestion(string $elementToDel, bool $default = false): ConfirmationQuestion
+    private function getDeleteConfirmationQuestion(array $elementToDel, bool $default = false): ConfirmationQuestion
```

Comment:
I'm not sure it is worth having a whole method only for creating a `ConfirmationQuestion`.
Also, you should use `$this->getAentHelper()->question('...')->yesNoQuestion()` as it actually checks that the answer is yes or no (Symfony ChoiceQuestion checks only for "y" and assumes anything else is a no).